build.sbt

```diff
@@ -11,10 +11,18 @@ Global / lintUnusedKeysOnLoad := false
 def dev(ghUser: String, name: String, email: String): Developer =
   Developer(ghUser, name, email, url(s"https://github.com/$ghUser"))
 
+lazy val commonSettings = Seq(
+  scalacOptions ++= {
+    if (tlIsScala3.value) Seq("-Xtarget:8") else Seq("-target:jvm-1.8")
+  },
+  javacOptions ++=
+    Seq("-target", "8", "-source", "8")
+)
+
```

Comment:
I'm pretty sure these are redundant, but if they are actually necessary then we should upstream these into sbt-typelevel since all projects should use it. WDYT?

Reply:
I have removed them, just switched the order of Scala versions in order to have Scala 3 the one with jvm17 and jvm8

Reply:
>but if they are actually necessary then we should upstream these into sbt-typelevel since all projects should use it. WDYT?
@armanbilge That would be a great solution. I intend to use sbt-typelevel for a project of mine that uses protobuf code generation where the class format version needs to be what 1.8 produces and CI uses temurin 17.

Reply:
Nice! Feel free to PR, it should go in the `TypelevelSettings` plugin.

Reply:
👍